2024-02-24 01:43:10,543 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-24 01:43:10,543 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Configure stats pid to 26232
2024-02-24 01:43:10,544 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-24 01:43:10,544 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-24 01:43:10,544 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-24 01:43:10,544 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-24 01:43:10,545 INFO    MainThread:26232 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240224_014310-jdatstd9\logs\debug.log
2024-02-24 01:43:10,545 INFO    MainThread:26232 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240224_014310-jdatstd9\logs\debug-internal.log
2024-02-24 01:43:10,545 INFO    MainThread:26232 [wandb_init.py:init():566] calling init triggers
2024-02-24 01:43:10,545 INFO    MainThread:26232 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'bootstrap': False, 'n_trees': 100, 'min_samples': 2, 'max_depth': 10, 'max_features': 'log2'}
2024-02-24 01:43:10,545 INFO    MainThread:26232 [wandb_init.py:init():591] re-initializing run, found existing run on stack: ao9dkfxh
2024-02-24 01:43:10,547 INFO    MainThread:26232 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/ao9dkfxh
2024-02-24 01:43:10,547 INFO    MainThread:26232 [jupyter.py:save_history():445] not saving jupyter history
2024-02-24 01:43:10,548 INFO    MainThread:26232 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-24 01:43:10,548 INFO    MainThread:26232 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-24 01:43:10,548 INFO    MainThread:26232 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-24 01:43:10,548 INFO    MainThread:26232 [wandb_run.py:_restore():2193] restore
2024-02-24 01:43:10,548 INFO    MainThread:26232 [wandb_run.py:_restore():2199] restore done
2024-02-24 01:43:17,538 INFO    MainThread:26232 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-24 01:43:17,539 INFO    MainThread:26232 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-24 01:43:17,545 INFO    MainThread:26232 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-24 01:43:17,744 INFO    MainThread:26232 [wandb_init.py:init():616] starting backend
2024-02-24 01:43:17,744 INFO    MainThread:26232 [wandb_init.py:init():620] setting up manager
2024-02-24 01:43:17,746 INFO    MainThread:26232 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-24 01:43:17,747 INFO    MainThread:26232 [wandb_init.py:init():628] backend started and connected
2024-02-24 01:43:17,754 INFO    MainThread:26232 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-24 01:43:17,754 INFO    MainThread:26232 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-24 01:43:17,755 INFO    MainThread:26232 [wandb_init.py:init():720] updated telemetry
2024-02-24 01:43:17,756 INFO    MainThread:26232 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-24 01:43:19,946 INFO    MainThread:26232 [wandb_run.py:_on_init():2262] communicating current version
2024-02-24 01:43:21,468 INFO    MainThread:26232 [wandb_run.py:_on_init():2271] got version response 
2024-02-24 01:43:21,469 INFO    MainThread:26232 [wandb_init.py:init():804] starting run threads in backend
2024-02-24 01:43:21,547 INFO    MainThread:26232 [wandb_run.py:_console_start():2241] atexit reg
2024-02-24 01:43:21,548 INFO    MainThread:26232 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-24 01:43:21,548 INFO    MainThread:26232 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-24 01:43:21,548 INFO    MainThread:26232 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-24 01:43:21,548 INFO    MainThread:26232 [wandb_init.py:init():847] run started, returning control to user process
2024-02-24 02:01:03,949 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-24 02:01:03,950 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Configure stats pid to 26232
2024-02-24 02:01:03,950 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-24 02:01:03,950 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-24 02:01:03,950 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-24 02:01:03,951 INFO    MainThread:26232 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-24 02:01:03,951 INFO    MainThread:26232 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240224_020103-rlznh432\logs\debug.log
2024-02-24 02:01:03,951 INFO    MainThread:26232 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240224_020103-rlznh432\logs\debug-internal.log
2024-02-24 02:01:03,951 INFO    MainThread:26232 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001C55A526950>
2024-02-24 02:01:03,952 INFO    MainThread:26232 [wandb_init.py:init():566] calling init triggers
2024-02-24 02:01:03,952 INFO    MainThread:26232 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'bootstrap': False, 'n_trees': 100, 'min_samples': 2, 'max_depth': 10, 'max_features': 'sqrt'}
2024-02-24 02:01:03,952 INFO    MainThread:26232 [wandb_init.py:init():591] re-initializing run, found existing run on stack: jdatstd9
2024-02-24 02:01:03,954 INFO    MainThread:26232 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/jdatstd9
2024-02-24 02:01:03,954 INFO    MainThread:26232 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-24 02:01:03,954 INFO    MainThread:26232 [wandb_run.py:_restore():2193] restore
2024-02-24 02:01:03,954 INFO    MainThread:26232 [wandb_run.py:_restore():2199] restore done
2024-02-24 02:01:11,326 INFO    MainThread:26232 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-24 02:01:11,327 INFO    MainThread:26232 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-24 02:01:11,333 INFO    MainThread:26232 [wandb_run.py:_footer_sync_info():3825] logging synced files
